collector.go

```diff
@@ -26,11 +26,13 @@ type cwCollectorTemplate struct {
 }
 
 type cwCollector struct {
-	Region            string
-	Target            string
-	ScrapeTime        prometheus.Gauge
-	ErroneousRequests prometheus.Counter
-	Template          *cwCollectorTemplate
+	Region                   string
+	Target                   string
+	ScrapeTime               prometheus.Gauge
+	ScrapeTimeBuckets        prometheus.Histogram
```

Comment:
Usually, we name those as Histograms
```suggestion
	ScrapeDurationHistogram  prometheus.Histogram
```

Reply:
Done :)